WMF Framework/WMFContentGroup+Extensions.m

```diff
@@ -637,13 +637,11 @@ - (nullable WMFContentGroup *)contentGroupForURL:(NSURL *)URL {
 }
 
 - (nullable WMFContentGroup *)newestGroupOfKind:(WMFContentGroupKind)kind requireIsVisible:(BOOL)isVisibleRequired {
```

Comment:
it looks like `isVisibleRequired` is no longer necessary or mistakenly ignored

Reply:
mistakenly ignored, fixed in 8c87c4649e5e879809b9717f2fdef18d1c7df68f